_posts/2023-11-23-announcing-dp-social-inbox.md

```diff
@@ -0,0 +1,101 @@
+---
+
+  
+### {{ page.excerpt }}
+  
+
+## Press Release
+
+Today, Hypha and [Sutty](https://sutty.nl/) are thrilled to announce the release of the Social Inbox, a new feature of Distributed.Press that integrates a website's comment section with federated social media platforms like Mastodon. With the Social Inbox enabled, websites obtain their own account on the Fediverse, allowing it to automatically send out new posts to followers at the time of publication. When other users reply to posts, you can approve them to be published to the site as comments. The Social Inbox allows readers to directly engage with your posts where they already are, and gives publishers the ability to incorporate public dialogue into their websites.
+
+## Failures of Existing Social Media
+
+Big social media services, like X/Twitter and Instagram, centralize all control over the data, content, and interactions on their platforms. They moderate messages on their platforms with little to no transparency nor accountability, leading to posts and entire accounts being censored with no explanation. Additionally, corporate social media platforms have an insatiable hunger for data. They surveil every aspect of our interactions - from what we post, what we "like", who we follow, to even minute details of where our attention lingers. This data is used to feed their algorithms in order to maximize the influence, and therefore profitability, of the ads they show to their users. 
+
+Their intrusive practices extend into the timeline itself, by manipulating what posts appear and inserting "suggested" content that we never ask to see. Major social media platforms regularly comply with government requests to hand over personal data without a court warrant. And we have witnessed how an entire social media network can be sold to a billionaire with no regard to the impact that would have on global public discourse. 
+
+## ActivityPub: A Decentralized Alternative
+
+In the face of the failures of existing social media platforms, there have been many attempts to build viable alternatives. Out of these, one of the most thriving networks is what is collectively called the Fediverse — an ecosystem of social media platforms that rely on the ActivityPub social data standard. 
+
+Instead of having a platform contain all the accounts, messages, and interactions all concentrated on one website, ActivityPub allows users with accounts on different websites to communicate with each other. Email works similarly in that you can send an email to someone else regardless of what email service they use. By enabling anyone to set up their own "instance" or social media server, ActivityPub allows people to decide who has control over all their data. They can decide to host their own instance, or choose to go with any of the many others who offer hosting services. The standard allows data to be hosted across thousands of different instances, while still being able to follow and send messages between them. 
```

Comment:
```suggestion
Instead of having a platform contain the accounts, messages, and interactions concentrated on one website, ActivityPub allows users with accounts on _different_ websites to communicate with each other. This is similar to how email works; you can send an email to someone else regardless of what email service they use. By enabling anyone to set up their own "instance" or social media server, ActivityPub allows people to decide who has control over their data. They can decide to host their own instance, or choose one from among the thousands of others who offer hosting services. The ActivityPub standard allows data to be hosted across thousands of different instances, while still being able to follow and send messages between them. 
```